core/src/com/fallenflame/game/PlayerModel.java

```diff
@@ -435,10 +439,14 @@ public void update(float dt) {
                 filmstrip.setFrame(frame + 1);
             } else if (walkCool > 0) {
                 walkCool--;
-            } else if (frame == filmstrip.getSize() - 1){
+            } else if (deathDelay <= 0){
                 life = LifeState.DEAD;
+            } else if (frame == filmstrip.getSize() - 1) {
+                deathDelay--;
             }
+
         } else if (isAlive()) {
+            animateFireBuddy(angle100);
```

Comment:
Also why is this changed?

Reply:
Fire Buddy would "animate" while player was dying. It would essentially just keep shaking side-to-side. Here, it does not move at all while player is dying